components/CharacterCard/CharacterCard.js

```diff
@@ -1 +1,25 @@
-export function CharacterCard() {}
+export default function createCharacterCard(character) {
+  const card = document.createElement("li");
```

Comment:
I suggest to use a different variable.

```suggestion
  const cardComponent = document.createElement("li");
```